gedcom/families.py

```diff
@@ -28,6 +29,7 @@ def __init__(self, people, validation_messages):
         self._people = people
         self._msgs = validation_messages
         self._current_time = datetime.now()
+        self._last_name_regex = re.compile("\/(.*)\/")
```

Comment:
Regex is great but sometimes it can be complicated to read. Can we add a comment to help others remember what this is doing